framework/core/js/src/forum/components/WelcomeHero.tsx

```diff
@@ -39,8 +39,8 @@ export default class WelcomeHero extends Component<IWelcomeHeroAttrs> {
           />
 
           <div class="containerNarrow">
-            <h2 class="Hero-title">{app.forum.attribute('welcomeTitle')}</h2>
-            <div class="Hero-subtitle">{m.trust(app.forum.attribute('welcomeMessage'))}</div>
+            <h1 class="Hero-title">{app.forum.attribute('welcomeTitle')}</h1>
+            <h2 class="Hero-subtitle">{m.trust(app.forum.attribute('welcomeMessage'))}</h2>
```

Comment:
same here

```suggestion
            <h1 class="Hero-title">{app.forum.attribute('welcomeTitle')}</h1>
            <div class="Hero-subtitle">{m.trust(app.forum.attribute('welcomeMessage'))}</div>
```

Reply:
I don't agree. Looking at the structure of the page, the next nested item (h3's) are user cards etc. I think if you have a WelcomeHero on the page then the sub title is definitely of a higher order than the content of each discussion listed. SEO is (mainly) about prioritizing content, subtitle - look at the class - is the second most important text.

Reply:
🤔 That doesn't make sense though, the hero subtitle is not a heading, it's the content after the hero heading, the content of the hero, a paragraph here would make more sense.

Subtitle being the class name is just naming.

Reply:
Subtitles also aren't really numbered headings. That causes confusion for accessibility. https://css-tricks.com/html-for-subheadings-and-headings/

Reply:
> the next nested item (h3's) are user cards etc

I think this is another thing that should be changed. Discussion titles on discussions list should use H2 header. And posts inside discussion should not have automatic headers at all, since there is no good one (poster name is terrible header in that case), and it is better to rely on users to use proper headers inside of posts.